Formula/p/poac.rb

```diff
@@ -19,15 +18,6 @@ class Poac < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "5861b764ec78792acd2aa3853b275b1eec71279d09cd659fa9432e9271d69d2a"
   end
 
-  depends_on "cmake" => :build
-  depends_on "boost"
-  depends_on "fmt"
-  depends_on "libgit2"
-  depends_on "openssl@3"
-  depends_on "spdlog"
-
-  uses_from_macos "libarchive"
```

Comment:
any PR ref for such change? (seems like a big dependency change)

Reply:
@chenrui333 Yes, here it is: https://github.com/poac-dev/poac/pull/799

Reply:
Thanks!